reviewnb_demo/demo.ipynb

```diff
@@ -85,71 +85,35 @@
        "      <td>6.0</td>\n",
```

Comment:
<p>number of rows are changed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/vyshaal/reviewnb-demo/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>